src/org/pahappa/systems/services/IncidentServiceImpl.java

```diff
@@ -44,7 +44,31 @@ public Incident saveIncident(Incident incident) throws Exception {
 
 	@Override
 	public Incident updateIncident(Incident incident) throws Exception {
-		// TODO Auto-generated method stub
+		for(Incident item: incidents){
+
+//			Check if the parameter Title matches the records in the array
```

Comment:
I think you can find a better way of doing this to have readable code.
- Use a function that takes in an intent and returns the same intent if it exists otherwise return a ValidationFailedExecption